src/components/JobLogs/withJobLogsSort.js

```diff
@@ -95,8 +95,9 @@ const withJobLogsSortComponent = WrappedComponent => {
           useFormatterFn,
           sortFn,
         } = this.sortColumns[sort];
+        const useCellFormatter = useFormatterFn && cellFormatter;
 
-        return sortDir * (useFormatterFn && cellFormatter ?
+        return sortDir * (useCellFormatter ?
```

Comment:
Let's write ternary operators like described in rules (point 9): https://wiki.folio.org/display/FOLIJET/Code+style+rules+not+covered+by+eslint